packages/office-ui-fabric-react/src/components/__snapshots__/Slider.Basic.Example.tsx.shot

```diff
@@ -80,6 +80,7 @@ exports[`Component Examples renders Slider.Basic.Example.tsx correctly 1`] = `
         aria-valuemax={5}
         aria-valuemin={1}
         aria-valuenow={2}
+        aria-valuetext="2"
```

Comment:
This seems redundant, as `aria-valuenow` is already defined. What value is this adding?

For reference, https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-valuenow_attribute suggests this is only required when `valuenow` does not accurately describe the slider.

Reply:
well, narrator wasn't reading aria-valuenow. Maybe it's expecting a string?

Reply:
aria-valuenow is typed to be a number, but it seems narrator doesn't read that number

Reply:
yeah, without the valuetext all i get is "Basic example, slider at..." and never says the number

![image](https://user-images.githubusercontent.com/1434956/69761871-7e5d7800-111d-11ea-878e-a582c4d6dde9.png)

Not sure if this is a narrator bug then, but the easy solution for us is to be explicit with the aria-valuetext

Reply:
same experience can be found here:

http://www.oaa-accessibility.org/examplep/slider1/

narrator won't read out the numeric 'valuenow' value.